diff -Naur grpc-1.26.0-original/src/python/grpcio_tests/tests/interop/_secure_intraop_test.py grpc-1.26.0/src/python/grpcio_tests/tests/interop/_secure_intraop_test.py
--- grpc-1.26.0-original/src/python/grpcio_tests/tests/interop/_secure_intraop_test.py	2019-12-17 20:29:27.000000000 -0500
+++ grpc-1.26.0/src/python/grpcio_tests/tests/interop/_secure_intraop_test.py	2021-03-15 10:09:54.802317262 -0400
@@ -13,6 +13,7 @@
 # limitations under the License.
 """Secure client-server interoperability as a unit test."""
 
+import os
 import unittest
 
 import grpc
@@ -29,6 +30,8 @@
 class SecureIntraopTest(_intraop_test_case.IntraopTestCase, unittest.TestCase):
 
     def setUp(self):
+        if 'FEDORA_NO_NETWORK_TESTS' in os.environ:
+            self.skipTest('Network tests disabled')
         self.server = test_common.test_server()
         test_pb2_grpc.add_TestServiceServicer_to_server(service.TestService(),
                                                         self.server)
diff -Naur grpc-1.26.0-original/src/python/grpcio_tests/tests/unit/_dns_resolver_test.py grpc-1.26.0/src/python/grpcio_tests/tests/unit/_dns_resolver_test.py
--- grpc-1.26.0-original/src/python/grpcio_tests/tests/unit/_dns_resolver_test.py	2019-12-17 20:29:27.000000000 -0500
+++ grpc-1.26.0/src/python/grpcio_tests/tests/unit/_dns_resolver_test.py	2021-03-15 10:08:08.159649736 -0400
@@ -13,6 +13,7 @@
 # limitations under the License.
 """Tests for an actual dns resolution."""
 
+import os
 import unittest
 import logging
 import six
@@ -45,6 +46,8 @@
     def tearDown(self):
         self._server.stop(None)
 
+    @unittest.skipIf('FEDORA_NO_NETWORK_TESTS' in os.environ,
+                     'Network tests disabled')
     def test_connect_loopback(self):
         # NOTE(https://github.com/grpc/grpc/issues/18422)
         # In short, Gevent + C-Ares = Segfault. The C-Ares driver is not
